.github/workflows/cd.yml

```diff
@@ -41,42 +52,43 @@ jobs:
         zip -r artifacts.zip artifacts
         echo "TAG_VERSION=${GITHUB_REF/refs\/tags\//}" >> $GITHUB_ENV
 
-    - name: Configure AWS Credentials
-      uses: aws-actions/configure-aws-credentials@v1
-      with:
-        aws-access-key-id: ${{ secrets.AWS_STAGING_ACCESS_KEY_ID }}
-        aws-secret-access-key: ${{ secrets.AWS_STAGING_SECRET_ACCESS_KEY }}
-        aws-region: us-west-2
-
-    - name: Upload Artifacts to S3
-      run: |
-        zip=`ls artifacts/*.zip`
-        rpm=`ls artifacts/*.rpm`
-        deb=`ls artifacts/*.deb`
-
-        # Inject the build number before the suffix
-        zip_outfile=`basename ${zip%.zip}-build-${GITHUB_RUN_NUMBER}.zip`
-        rpm_outfile=`basename ${rpm%.rpm}-build-${GITHUB_RUN_NUMBER}.rpm`
-        deb_outfile=`basename ${deb%.deb}-build-${GITHUB_RUN_NUMBER}.deb`
-
-        s3_prefix="s3://staging.artifacts.opendistroforelasticsearch.amazon.com/snapshots/elasticsearch-plugins/security/"
-
-        echo "Copying ${zip} to ${s3_prefix}${zip_outfile}"
-        aws s3 cp --quiet $zip ${s3_prefix}${zip_outfile}
-
-        echo "Copying ${rpm} to ${s3_prefix}${rpm_outfile}"
-        aws s3 cp --quiet $rpm ${s3_prefix}${rpm_outfile}
-
-        echo "Copying ${deb} to ${s3_prefix}${deb_outfile}"
-        aws s3 cp --quiet $deb ${s3_prefix}${deb_outfile}
-
-    - name: Upload Artifacts to Maven Central
-      env:
-        SONATYPE_USER: ${{ secrets.SONATYPE_USER }}
-        SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
-      run: |
-        gpg --batch --import --no-tty <(echo -e "${{ secrets.PGP_PRIVATE_KEY }}")
-        mvn -B deploy -Padvanced -Prelease -DskipTests -Dgpg.passphrase=${{ secrets.PGP_PASSPHRASE }}
+# TODO: Update the following configuration commented out once the new CI/CD process of OpenSearch is confirmed.
```

Comment:
Delete. It is not clear what will be the approach to publish to maven. In any case we have source control if this needs to be restored.

Reply:
Done